eel/src/storage_persister.rs

```diff
@@ -318,22 +318,9 @@ impl StoragePersister {
     }
 
     pub fn fetch_remote_channel_manager_serialized(&self) -> Result<Vec<u8>> {
-        let encrypted_data = match self
+        let encrypted_data = self
             .storage
-            .get_object(OBJECTS_BUCKET.to_string(), MANAGER_KEY.to_string())
-        {
-            Ok(data) => data,
-            Err(RuntimeError {
-                code: RuntimeErrorCode::ObjectNotFound,
-                ..
-            }) => {
-                return Err(runtime_error(
-                    RuntimeErrorCode::NonExistingWallet,
-                    "Failed to find remote ChannelManager",
-                ))
-            }
-            Err(e) => return Err(e),
-        };
+            .get_object(OBJECTS_BUCKET.to_string(), MANAGER_KEY.to_string())?;
```

Comment:
If the recovery method fails, how can the consumer distinguish between a non-existing wallet and the remote storage service being unavailable?

Reply:
The idea was that the `RemoteStorage::get_object()` implementation would return a specific error when the remote storage could be reached, but it reported no such object. Looking at codebase at `main` I think we had a bug as we only used `ObjectNotFound` in the file storage implementation, and not on the proper remote storage implementation (`eel_interface_impl.rs`).